web/src/main/java/org/cbioportal/genome_nexus/web/AnnotationController.java

```diff
@@ -215,7 +215,7 @@ public VariantAnnotation fetchVariantAnnotationGET(
     public VariantAnnotation fetchVariantAnnotationByGenomicLocationGET(
         @ApiParam(value="A genomic location. For example 7,140453136,140453136,A,T",
             required = true)
-        @PathVariable @ValidGenomicLocation String genomicLocation,
+        @PathVariable String genomicLocation,
```

Comment:
is this `@ValidGenomicLocation` annotation used anywhere else?

If it's unused anywhere else in the code then perhaps we can remove the class completely

Reply:
It's not used elsewhere, thinking of keeping it until someone more familiar with the code approves. Seems like this may have been purposely introduced to prevent incorrectly formatted from being submitted (but I guess that is the behavior we are changing)

Reply:
`@ValidGenomicLocation` does a pre-validation for this field. So now we want to query the VEP with an invalid genomic location even if we are sure that it's going to fail?